providers/atlassian/connection/admin/connection.go

```
@@ -0,0 +1,82 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package admin
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ctreminiom/go-atlassian/admin"
+	"github.com/rs/zerolog/log"
+	"go.mondoo.com/cnquery/v9/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnquery/v9/providers/atlassian/connection/shared"
+)
+
+const (
+	Admin shared.ConnectionType = "admin"
+)
+
+type AdminConnection struct {
+	id     uint32
+	Conf   *inventory.Config
+	asset  *inventory.Asset
+	client *admin.Client
+	host   string
+}
+
+func NewConnection(id uint32, asset *inventory.Asset, conf *inventory.Config) (*AdminConnection, error) {
+	adminToken := conf.Options["admintoken"]
+	if adminToken == "" {
+		adminToken = os.Getenv("ATLASSIAN_ADMIN_TOKEN")
+	}
+	if adminToken == "" {
+		return nil, errors.New("you need to provide atlassian admin token via ATLASSIAN_ADMIN_TOKEN env")
+	}
+
+	client, err := admin.New(nil)
+	if err != nil {
+		log.Fatal().Err(err)
```

let's return the error here